mava/wrappers/jumanji.py

```diff
@@ -217,3 +220,128 @@ def observation_spec(self) -> specs.Spec[Union[Observation, ObservationGlobalSta
             )
 
         return spec
+
+
+class MultiCVRPWrapper(Wrapper):
+    """Wrapper for MultiCVRP environment."""
+
+    def __init__(self, env: MultiCVRP, has_global_state: bool = False):
+        self.num_agents = env._num_vehicles
+        self._env = env
+        self.has_global_state = has_global_state
```

Comment:
Missing the call to `super`

```suggestion
    def __init__(self, env: MultiCVRP, has_global_state: bool = False):
        super().__init__(env)
        self._env: MultiCVRP
        
        self.num_agents = env._num_vehicles
        self.has_global_state = has_global_state
```